cms/server/util.py

```diff
@@ -493,7 +493,7 @@ def format_token_rules(tokens, t_type=None, locale=None):
         tokens["type_pl"] = _("tokens")
 
     tokens["min_interval"] = tokens["min_interval"].total_seconds()
-    tokens["gen_interval"] = tokens["gen_interval"].total_seconds() / 60
+    tokens["gen_interval"] = tokens["gen_interval"].total_seconds() // 60
```

Comment:
You sure about this? total_seconds() should be a float, so before it was float division - I think.

Reply:
It was float division, but the only uses of the result are comparing it to 1.0 and then formatting it with `%d`: I think it's better if the division returns an int.

Reply:
I think I see a print with %g, and no comparison. You looking at the right one?

Reply:
Oh, yes, you're right, I must have misread, I changed this back to a float division.